volmdlr/wires.py

```diff
@@ -2884,31 +2867,48 @@ def select_closest_sewing_closing_point(self,
                 dist = intr_list[0].point_distance(line_segment.start)
                 if dist < distance:
                     distance = dist
-                    if intr_list[0].point_distance(intr_list[1].start) < \
-                            intr_list[0].point_distance(intr_list[1].end):
-                        closing_point = intr_list[1].start
-                    else:
-                        closing_point = intr_list[1].end
+                    closing_point = self.choose_closing_point(intr_list[0],
+                                                              intr_list[1])
             elif intr_list[0] == middle_point and \
                     polygon_primitive.length() == intr_list[1].length():
                 closing_point = intr_list[1].start
                 distance = 0
 
         return closing_point
 
-    def search_farthest(self, point, possible_closing_points):
+    def search_farthest(self, interseting_point, possible_closing_points):
+        """
+        While Sewing two Polygons, and searching a face\'s closing point, this
+        method verifies it shoul the closest of the farthest available
+        :return: True if to search the farthest of False if not
+        """
         distance = math.inf
         target_prim = None
-        for i, (intersection_point, prim) in enumerate(possible_closing_points):
-            dist = point.point_distance(intersection_point)
+        for intersection_point, prim in possible_closing_points:
+            dist = interseting_point.point_distance(intersection_point)
             if dist < distance:
                 distance = dist
                 target_prim = prim
         if target_prim in self.line_segments:
             return True
         return False
 
+    @staticmethod
+    def choose_closing_point(intersection_point, primitive):
+        """
+        Given an intersection point, The methods verifies if
+        closing points start or end, whichever the closest one
+        :param intersection_point:
+        :param primitive:
+        :return: closing point
+        """
+        if intersection_point.point_distance(primitive.start) < \
+                intersection_point.point_distance(primitive.end):
+            return primitive.start
+        return primitive.end
```

Comment:
Should it really be a method ? 
It's just 1 line of code : `return (primitive.start if intersection_point.point_distance(primitive.start) < intersection_point.point_distance(primitive.end) else primitive.end)`

Reply:
It was just that I was doing the same thing a few times throughout the code, so I thought it would be a good idea to make it a separate method. But you're right, when we put it in a single line of code it is not really a big deal. Thank you, I'll fix it.

Reply:
Done